examples/common_vmseries/example.tfvars

```diff
@@ -22,7 +22,7 @@ vnets = {
             direction                  = "Inbound"
             access                     = "Allow"
             protocol                   = "Tcp"
-            source_address_prefixes    = ["1.2.3.4"]
+            source_address_prefixes    = ["134.238.135.14", "134.238.135.140"]
```

Comment:
We sure we want to add our public IPs into the examples? I'm more into adding `0.0.0.0/0` with a comment for the user to update this one with his own values.

Reply:
yeah, you are right, although I was not paying that much attention to examples yet